src/main/java/frc/robot/flywheel/FlywheelSubsystem.java

```diff
@@ -13,34 +15,40 @@ public class FlywheelSubsystem extends SubsystemBase {
      * 1 Follower TalonFX (Right)
      * Total: 2 TalonFX motors
      */
-
+    TalonFX adi;
```

Comment:
Change your motor names to leftMasterMotor and rightFollowerMotor or names that clearly show what the variables are created for